### PR TITLE
SidebarMenu 

### PR DESCRIPTION
**Pull request by Kluzko**

### CODE REVIEW

File: src/components/Cards/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import Image from "next/image";
+
+type CardProps = {
+  title: string;
+  img: string;
+  href: string;
+  name: string;
+};
+
+// Instead of dangerouslySetInnerHTML i choose to use more secure solution with React Markdown
```
Comment (Sumick) [inline src/components/Cards/Card.tsx:12]: **praise:** Good idea :-)

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import styles from "styles/footer.module.css";
+import { AiOutlineGithub, AiOutlineLinkedin } from "react-icons/ai";
+import { MdWeb } from "react-icons/md";
+
+import { LogoNerdbord } from "./LogoNerdbord";
+
+const Socials = [
+  {
```
Comment (Sumick) [inline src/components/Footer.tsx]: **praise:** Smart move :D

File: src/components/Cards/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import styles from "styles/card.module.css";
```
Comment (Sumick) [inline src/components/Cards/Card.tsx]: **nitpick:** Component styles should be placed together with related component

File: src/components/Cards/index.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import styles from "styles/cards.module.css";
+import { Card } from "./Card";
+import { MenuList } from "data";
+
+export const Cards = () => (
```
Comment (Sumick) [inline src/components/Cards/index.tsx]: **nitpick:** This is a list - it is worth to pass data as props here

File: src/components/Cards/index.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  <div className={styles.container}>
```
Comment (Sumick) [inline src/components/Cards/index.tsx]: **nitpick:** If this is a list, why not to use `ul` to keep good semantics?

File: src/components/Menu/index.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
+import { useState, useRef } from "react";
+import styles from "styles/menu.module.css";
+import { MenuItem } from "./MenuItem";
+import { MenuList } from "data";
+import { useGetActiveSection, useOnOuterClick } from "hooks";
+
+export const Menu = () => {
+  const [isActive, setIsActive] = useState(false);
+  const ref = useRef(null);
```
Comment (Sumick) [inline src/components/Menu/index.tsx]: **nitpick:** Ref element could be typed

File: src/hooks/useGetActiveSection.ts [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+type MenuListProps = typeof MenuList;
+
+export const useGetActiveSection = (menuList: MenuListProps) => {
+  const [activeMenuItem, setActiveMenuItem] = useState("");
+  const handleScroll = useCallback(() => {
+    const offset = 400;
+    const scrollPosition = window.scrollY + offset;
+
+    const activeItem = menuList.find((item) => {
+      const section = document.querySelector(item.href) as HTMLElement;
```
Comment (Sumick) [inline src/hooks/useGetActiveSection.ts]: **nitpick:** We should avoid type casting like this

**Review by Sumick — APPROVED** [review]
Your code looks good, my comments are non-blocking ones. I will try to get back to you until the end of the current work week.

**Comment by Kluzko** [comment]
Hi , thanks for reviewing my pull request , I have applied suggested changes. ⠀⠀⠀⠀⠀

**Pull request merged**